services/notifications/pkg/email/email.go

```diff
@@ -77,3 +105,66 @@ func executeTemplate(tpl *template.Template, vars any) (string, error) {
 	}
 	return writer.String(), nil
 }
+
+func readFs() (map[string][]byte, error) {
+	dir := filepath.Join("templates", "html", "img")
+	entries, err := templatesFS.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	list := make(map[string][]byte)
+	for _, e := range entries {
+		if !e.IsDir() {
+			file, err := templatesFS.ReadFile(filepath.Join(dir, e.Name()))
+			if err != nil {
+				return nil, err
+			}
+			if !validateMime(file) {
+				continue
+			}
+			list[e.Name()] = file
+		}
+	}
+	return list, nil
+}
+
+func readImages(emailTemplatePath string) (map[string][]byte, error) {
+	dir := filepath.Join(emailTemplatePath, "html", "img")
+	entries, err := os.ReadDir(dir)
```

Comment:
`os.ReadDir` makes it unflexible. Use `os.DirFS`. Move common code from `readFS` and `readImages` in a common function taking a `fs.FS`

Reply:
done